.github/actions/performance-tests/art.yaml

```
@@ -58,13 +58,13 @@ scenarios:
+              as: response
           ensure:
-            - statusCode: 200
-            - statusCode: 201
-          assert:
-            - subject: response.error
-              equals: null
+            - statusCode:
+                - 200
+                - 201
+          expect:
+          - notHasProperty: "{{ response.error }}"
```

yamllint

[warning] wrong indentation: expected 12 but found 10

(indentation)